src/Core/src/Handlers/Entry/EntryHandler.Android.cs

```diff
@@ -30,47 +27,40 @@ protected override AppCompatEditText CreateNativeView()
 
 		// Returns the default 'X' char drawable in the AppCompatEditText.
 		protected virtual Drawable GetClearButtonDrawable() =>
-			ContextCompat.GetDrawable(Context, Resource.Drawable.abc_ic_clear_material);
+			_clearButtonDrawable ??= ContextCompat.GetDrawable(Context, Resource.Drawable.abc_ic_clear_material);
```

Comment:
Cache the value instead of using the Defaults system.

Reply:
Can we make this lazy?  I think because this is retrieved as part of the mapper it'll get retrieved with every single entry control on inception but we only really need this when the clearbutton is going to be visible

I also wonder if we can just make this a static inside the entryextensions

I don't recall how weird it is to make a drawable static and reuse it... I think if you wrap/mutate it then you can make tinting changes without affecting the root drawable

Reply:
nm I don't think mutate/static will work here

Either way I don't think it's currently lazy enough :-)